From 1714046ef08ef685bcf1d6c154e64809b12e8d7f Mon Sep 17 00:00:00 2001
From: Tiago Vignatti <tiago.vignatti@nokia.com>
Date: Fri, 18 Sep 2009 15:21:19 +0300
Subject: [PATCH 4/4] vgaarb: fix newbie typo

Eh, we don't need to check for the fd anyway. Kudos jcristau for reporting.

Signed-off-by: Tiago Vignatti <tiago.vignatti@nokia.com>
---
 src/common_vgaarb.c |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/src/common_vgaarb.c b/src/common_vgaarb.c
index f1c71e0..6358a66 100644
--- a/src/common_vgaarb.c
+++ b/src/common_vgaarb.c
@@ -153,9 +153,6 @@ pci_device_vgaarb_fini(void)
     if (!pci_sys)
         return;
 
-    if (!pci_sys->vgaarb_fd)
-        return;
-
     close(pci_sys->vgaarb_fd);
 }
 
-- 
1.6.5.rc1

